shared/src/main/scala/coop/rchain/grpcmonix/GrpcMonix.scala

```diff
@@ -64,10 +69,13 @@ object GrpcMonix {
   def grpcObserverToMonixCallback[T](observer: StreamObserver[T]): Callback[T] =
     new Callback[T] {
       override def onError(t: Throwable): Unit = observer.onError(t)
-      override def onSuccess(value: T): Unit = {
-        observer.onNext(value)
-        observer.onCompleted()
-      }
+      override def onSuccess(value: T): Unit =
+        try {
+          observer.onNext(value)
+          observer.onCompleted()
+        } catch {
+          case t: Throwable => logger.warn(s"Failed to send response: ${t.getMessage}")
```

Comment:
What is going to happen in the other method - `onError`? Shouldn't we have identical handling there?

Reply:
I was considering to apply the same handling here and catch only `NonFatal` exceptions. However, the Monix `Obeserver` contract states that we shouldn't throw any exceptions inside `onNext`

Reply:
I mean not `onNext` but `onError`. There we also notify `observer` about something. In this case it's an error.